src/components/PostPageDetail/index.js

```diff
@@ -586,7 +587,6 @@ const PostPageDetailIdComponent = (props) => {
               setItem({...item, is_following_target: !item?.is_following_target});
             }}
           />
-
           <ScrollView
             ref={scrollViewRef}
             keyboardShouldPersistTaps="handled"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change seems fine, but be aware of potential re-rendering issues. The `setItem` function updates the state and causes a re-render. If the `is_following_target` property changes frequently, it could lead to performance issues due to excessive re-rendering.
```diff
- setItem({...item, is_following_target: !item?.is_following_target});
+ setItem(prevItem => ({...prevItem, is_following_target: !prevItem?.is_following_target}));
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->